### DISCUSSION TITLE
remove left navigation bar e static top bar

### DISCUSSION THREAD

**Discussion by tmartinscopo** [serenity-is/Serenity#7179] [Help]
Hello,

I'm using the MVC version of Serenity on mobile devices. Is it possible on a page to completely remove the left navigation bar and the static top bar (marked in yellow in the image)?

![image](https://github.com/serenity-is/Serenity/assets/37545074/a6552b00-b4d4-4118-b633-621cd3f96132)

Thank You

**Comment by LSNicholls** [comment]
I am probably not using the same version of Serenity you are using, so this might not help but... if I wanted to do this, I would be looking at _Sidebar.cshtml.  Does this file exist in your version?
